packages/graphql/src/schema/resolvers/wrapper.ts

```diff
@@ -19,12 +19,14 @@
 
 import Debug from "debug";
 import { GraphQLResolveInfo, GraphQLSchema, print } from "graphql";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { JwtPayload } from "jsonwebtoken";
```

Comment:
mmm, I think this may break when creating the package as jsonwebtoken will not be in the required dependencies.

If we only care about the type JwtPayload, I'd consider just copying it to our codebase

Reply:
I may be wrong, but I think this will just fail to execute when running in a production environment as we are importing the real library `jsonwebtoken`

WE either move `jsonwebtoken` to dependencies (@types can be in devDependencies) or copy the type (wich doesn't seem to be a huge type)

Reply:
I've copied it!